src/frontend/src/lib/styles/theme/base-colors.scss

```diff
@@ -0,0 +1,7 @@
+$alice-blue: #e8f1ff;
+$water: #d1e3ff;
+$pale-cornflower-blue: #b0cdff;
+$french-sky-blue: #84b2ff;
+$crayola: #004abe;
+$catalina-blue: #012f80;
+$brandeis-blue: #0066ff;
```

Comment:
ah! is it not `blue-ribbon`? I swear I thought it was

Reply:
I'm using whatever pops out here :D
<img width="455" alt="image" src="https://github.com/user-attachments/assets/bac50fd4-cee0-409a-8a94-a929aaed490e">

Reply:
perfect then! I must have made a mistake at start eheh